implementation.md

```diff
@@ -25,3 +25,60 @@ Implementations that support the Transactions Extension or require credentials f
 implement different behavior, for example, allowing credentials when requests are coming from a trusted domain, 
 allowing DELETE, PUT, or PATCH methods, or 
 permitting the `If-Match` request header.
+
+## datetime Parameter Handling
+
+The datetime parameter used by the Item Search and Features conformance classes allows the same values as the 
+[OAF datetime](http://docs.opengeospatial.org/is/17-069r3/17-069r3.html#_parameter_datetime) parameter. This allows for 
+either a single [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339) datetime or an open or closed interval that also 
+uses RFC 3339 datetimes.
+
+RFC 3339 is a profile of ISO 8601, adding these constraints to the allowed values:
+- a complete representation of date and time (fractional seconds optional).
+- requires 4-digit years
+- only allows a period character to be used as the decimal point for fractional seconds
+- requires the zone offset to be `Z` or like `+00:00`, while ISO8601 allows timezones like `+0000`
+
+A simple regex for an RFC 3339 datetime is:
+
+```
+^(\d\d\d\d)\-(\d\d)\-(\d\d)(T|t)(\d\d):(\d\d):(\d\d)([,.]\d+)?(Z|([-+])(\d\d):(\d\d))$
+```
+
+This is not a precise regex, as it matches some strings that violate semantics. There are additional restrictions, for example, 
+the month (01-12), the day (01-31), the hour (0-24), minute (0-60), seconds (0-9), and timezone offsets.  However, the best 
+strategy is to use this regex to ensure the datetime conforms to the the RFC3339 profile and then us an ISO8601 parser to produce
+a valid datetime object from the datetime string.
+
+Thereby, the recommended process for parsing the datetime is:
+
+1. uppercase the string (this avoids needing to match on both (t|T) and (z|Z))
+2. split the string on `/` to determine if it is a single datetime or an interval
```

Comment:
The interval semantic was not mentioned before and it's not 100% clear whether the regexp accounts for this case (it does not, but should it?)

Reply:
I'm clarified the text. The regex is only for parsing a single datetime value.